nebula_windows.go

```
@@ -0,0 +1,159 @@
+package nerf
+
+import (
+	"fmt"
+	"net"
+	"os/exec"
+	"path"
+	"strings"
+
+	"go.uber.org/zap"
+)
+
+func nebulaDownloadLink() string {
+	return "https://github.com/hostinger/packages/releases/download/v1.0.0/nebula-1.5.2-windows-amd64.exe"
+}
+
+// NebulaDir absolute paths to the directory of Nebula configurations and binaries
+func NebulaDir() string {
+	return "C:/Nebula"
+}
+
+// NebulaExecutable show full path of Nebula executable
+func NebulaExecutable() string {
+	return path.Join(NebulaDir(), "nebula")
+}
+
+func getDefaultGateway(x string) (string, error) {
+	args := x
+	var defaultGateway string
+	lines, err := exec.Command("powershell", "Find-NetRoute", "-RemoteIPAddress", args).CombinedOutput()
+	if err != nil {
+		return "", err
+	}
+	for _, param := range strings.Split(string(lines), "\n") {
+		if !strings.Contains(param, "NextHop") {
+			continue
+		}
+		defaultGateway = strings.TrimSpace(strings.Split(param, ":")[1])
```

You should check if the split didn't fail, otherwise, you will have unknowns.